server/package.json

```diff
@@ -51,7 +51,6 @@
     "figlet": "^1.8.0",
     "hedera-agent-kit": "^1.0.3",
     "hono": "^4.6.12",
-    "move-agent-kit": "^0.2.0",
```

Comment:
untill the package bug is fixed , this looks good to do

Reply:
yes